### DISCUSSION TITLE
真·二次元纸片人射击游戏它来啦！

### DISCUSSION THREAD

**Discussion by StrinovaMaster** [share121/inter-knot#3403] [其他]
击穿次元 多维枪战 
卡拉彼丘 启动！
官网链接（下载地址）:https://klbq.qq.com/
![82e49756c55824cb](https://github.com/user-attachments/assets/93ca091a-a69e-4d41-b56b-655a44fafab1)

**Comment by Mirrorsing** [comment]
这咋还有真打广告的，旧世纪的游戏了现在还在运营吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Aha-SWYH** [comment]
？绳网团队不管管吗

**Comment by StrinovaMaster** [comment]
孩子们其实我装成官方打广告是为了流量（游戏内的绳网也有很多广告）

**Reply by Mirrorsing** [reply]
原来是这样吗，我还以为是真的有旧世纪的游戏能运营到现在呢哈哈，不过仔细想想如果真的存在这样的公司的话那也算是新艾利都的一个传奇了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/379fc4175e2c143f8c5d5d55e8bc7f5a_1069357194481574423.png" alt="猫又-好喜欢" width="75" >

**Reply by Wax1234567** [reply]
原来是这样吗，演的太像了点吧。不过有一说一，用真实存在的游戏是否有点不妥<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >

**Comment by momomomofish** [comment]
byd卡拉比丘还在追我
